src/eq_fault_geom/geomio/cfm_faults.py

```diff
@@ -28,6 +29,44 @@
 # List of subduction zones to exclude if desired
 subduction_names = ("hikurangi", "puysegur")
 
+domain_names = ['Alpine Fault', 'Marlborough Fault System', 'Central Canterbury',
```

Comment:
@andy22b  just checking that need both these data structs? It looks to me like the list is superceded by the dict. I guess this means that the mapping defined in the CFM itself?